spec/migrations/20191112132759_copy_owner_id_to_group_id_for_dashboards_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_migration
+
+describe CopyOwnerIdToGroupIdForDashboards do
+  let(:miq_set)  { migration_stub(:MiqSet) }
+  let(:name)     { "Hello Dashboard" }
+  let(:group_id) { 1 }
+
+  migration_context :up do
+    it "Copying owner_id to group_id in miq_sets table for each dashboard assigned to group only" do
+      dashboard = miq_set.create!(:name => name, :set_type => "MiqWidgetSet", :owner_type => "MiqGroup", :owner_id => group_id)
```

Comment:
Can you also create examples of non-MiqWidgetSet rows, and ensure they are not modified?

Reply:
ok